concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+RonaTas-2011-431-1	1	carpenter	1042	CARPENTER
+RonaTas-2011-431-2	2	cannon, catapult	3202	CANNON
+RonaTas-2011-431-3	3	to fondle, pet, caress	3780	PET
+RonaTas-2011-431-4	4	stream, current	2935	CURRENT (STREAM)
+RonaTas-2011-431-5	5	false, imitation	3069	FALSE
+RonaTas-2011-431-6	6	pied (of an animals coat)		
+RonaTas-2011-431-7	7	apple	1320	APPLE
+RonaTas-2011-431-8	8	barrel (wooden container andstandard quantity)	2548	BARREL
+RonaTas-2011-431-9	9	to nurse, take care of	2472	SUCKLE (BREASTFEED, NURSE)
+RonaTas-2011-431-10	10	to decay, putrify, turn stale	2907	ROT
+RonaTas-2011-431-11	11	small, tiny	1246	SMALL
+RonaTas-2011-431-12	12	to mow	190	MOW
+RonaTas-2011-431-13	13	to decay, putrefy, go bad (of food, water)	2907	ROT
+RonaTas-2011-431-14	14	to harm	44	INJURE
+RonaTas-2011-431-15	15	a type of lasso		
+RonaTas-2011-431-16	16	ditch, canal	2002	DITCH
+RonaTas-2011-431-17	17	barley, Hordeum	932	BARLEY
+RonaTas-2011-431-18	18	barrow	2565	CASTRATED BOAR
```

Comment:
I didn't knew the word "barrow", but it seems to be the german "Hügelgrab" or  "Schubkarre". So I'd unmap.

Reply:
I agree. I found this description for looking up "Old English barrow": Barrows are useful for carrying loads of things, like dirt or leaves or sand, from one place to another.

Reply:
Oh, there is a Hungarian original? Then it could even make sense to add this as an extra column. Or do the authors provide the English concept as well?

Reply:
Yeah, this is how the entry looks like:

![Screenshot from 2022-10-11 12-01-36](https://user-images.githubusercontent.com/53435298/195063211-ceddeee3-3ed0-48fe-845a-359a80b8f595.png)

![Screenshot from 2022-10-11 12-01-50](https://user-images.githubusercontent.com/53435298/195063192-fa224514-f48d-4b7d-b58d-0873e14097e7.png)

So they provide the translation already, but when in doubt you can always look up the Hungarian word in some other dictionary as well (like I did with dict.cc and wiktionary in the previous comment)

Reply:
Idk, should I add an extra column with the Hungarian words? Haven't thought about that yet.

Reply:
I am not sure, as they also provide the translation themselves. On the other hand, it clearly helps disambiguating the cases. @AnnikaTjuka what do you think?

Reply:
I assumed it is a concept list, not an etymological items list (similar to [Zhang-2019-308](https://concepticon.clld.org/contributions/Zhang-2019-308)). I see that adding the list has already been discussed in several issues, so I would like to ask @LinguList to make the final decision here.

Reply:
Let us leave out the Hungarian for now. It will be available in the lexibank data, and in this way, we really reduce it to a concept list -- or hope to do so.

Reply:
Alright, then I will close this convo now.